frontend/utilities/endpoints.ts

```diff
@@ -65,8 +65,8 @@ export default {
   MACADMINS: `/${API_VERSION}/fleet/macadmins`,
 
   // MDM endpoints
-  MDM_APPLE: `/${API_VERSION}/fleet/mdm/apple`,
```

Comment:
these endpoints (`mdm/apple` and `mdm/apple_bm` are deprecated and now `/apns` and `/abm` respectively